packages/core/src/lib/waku_store/index.ts

```diff
@@ -370,3 +377,20 @@ async function* paginate<T extends DecodedMessage>(
 export function isDefined<T>(msg: T | undefined): msg is T {
   return !!msg;
 }
+
+export async function createCursor(
+  message: string,
```

Comment:
is that supposed to be the `payload` of a message? If so, it should be a `Uint8Array`.

I suggest passing `DecodedMessage` here because that's what the API user should use:

1. a message `M` is received (type is `DecodedMessage`
2. Device goes offline
3. Device goes back online, user wants to retrieve messages after `M`
4. Create cursor from `M`, do store query.